tck-sectionid-renaming.xsl

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source
+  ~ Copyright 2012, Red Hat, Inc. and/or its affiliates, and individual contributors
```

Comment:
Yeah, I plan to do a clean up in the spec regarding that. I chose to be consistent and change everything at the same time.